pkg/controllers/subnetset/vpc_handler.go

```diff
@@ -3,26 +3,24 @@ package subnetset
 import (
 	"context"
 
-	"github.com/vmware-tanzu/nsx-operator/pkg/nsx/services/common"
-
-	"sigs.k8s.io/controller-runtime/pkg/predicate"
-
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/util/retry"
 	"k8s.io/client-go/util/workqueue"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/event"
+	"sigs.k8s.io/controller-runtime/pkg/predicate"
 
 	"github.com/vmware-tanzu/nsx-operator/pkg/apis/v1alpha1"
+	"github.com/vmware-tanzu/nsx-operator/pkg/nsx/services/common"
 )
 
 // VPCHandler handles VPC event for SubnetSet:
 // - VPC creation: create default SubnetSet for the VPC.
 // - VPC deletion: delete all SubnetSets under the VPC.
 
 var defaultSubnetSets = map[string]string{
-	"default-vm-subnetset":  common.LabelDefaultVMSubnetSet,
-	"default-pod-subnetset": common.LabelDefaultPodSubnetSet,
+	"vm-default":  common.LabelDefaultVMSubnetSet,
```

Comment:
To align with this, I think the name [here](https://github.com/vmware-tanzu/nsx-operator/blob/main/test/e2e/nsx_subnet_test.go#L16) also needs to be updated. So maybe it's better to move this to common.types like `common.XXXXXX`.

Reply:
done